packages/trie/README.md

```diff
@@ -155,66 +185,47 @@ The `createProof` and `verifyProof` functions allow you to verify that a certain
 The following code demonstrates how to construct and subsequently verify a proof that confirms the existence of the key `test` (which corresponds with the value `one`) within the given trie. This is also known as inclusion, hence the name 'Proof-of-Inclusion.'
 
 ```ts
-import { Trie } from '@ethereumjs/trie'
-import { bytesToUtf8, utf8ToBytes } from '@ethereumjs/util'
-
-const trie = new Trie()
-
-async function test() {
-  await trie.put(utf8ToBytes('test'), utf8ToBytes('one'))
-  const proof = await trie.createProof(utf8ToBytes('test'))
-  const value = await trie.verifyProof(trie.root(), utf8ToBytes('test'), proof)
-  console.log(value ? bytesToUtf8(value) : 'not found') // 'one'
-}
+// ./examples/proofs.ts#L12-L16
 
-test()
+// proof-of-inclusion
+await trie.put(t1, v1)
```

Comment:
The `t1, v1` values are defined in the example but I do not see them here? Is my rendering wrong?

Reply:
![image](https://github.com/ethereumjs/ethereumjs-monorepo/assets/29359032/30648291-2e6e-4d09-9e0b-da3065c3f8c6)
I get this when I check GH

Reply:
If you look at the `// ./examples/proofs.ts#L...`, we're just doing a snippet here, I would be of the opinion that this is okay if we add some explanatory comments on what the values are.  Otherwise, we are repeating the example in full multiple times.

Reply:
I've switched all variable names to reflect key-values, so it's now `k1` and `v1` instead. This should rely on the classic key-value convention of naming and the details of what exactly the values assigned are can be omitted.